core/templates/services/UpgradedServices.ts

```diff
@@ -910,6 +903,9 @@ export class UpgradedServices {
         upgradedServices['baseInteractionValidationService']);
     upgradedServices['SkillCreationBackendApiService'] =
         new SkillCreationBackendApiService(upgradedServices['HttpClient']);
+    upgradedServices['SpeechSynthesisChunkerService'] =
```

Comment:
@donosco98 Can you add these new services to a file called `oppia-angular-root.component.ts`. you can find the steps for the same [here](https://github.com/oppia/oppia/wiki/Angular-Migration#implementation-details-to-migrate-services), under point number 12.

Reply:
Hi @srijanreddy98 , `SpeechSynthesisChunkerService` and `htmlEscaperService` are already added in the file you mentioed.

Reply:
Oh, in that case, I have no further concerns.